src/Layout/PresentationMenu/PresentationSettings/PresentationSettings.tsx

```diff
@@ -226,7 +227,7 @@ export class PresentationSettingsMenu extends React.Component<PresentationSettin
                                 changeDisproportionalityIndex={this.props.changeDisproportionalityIndex}
                                 disproportionalityIndex={this.props.disproportionalityIndex}
                                 tooltip={
-                                    <TooltipInfo
+                                    <TooltipInfoBottom
                                         text={"Her kan du velge mellom Loosemore-Hanbys (L-H) og Gallaghers (LSq)."}
```

Comment:
The ordering of these might benefit from being swapped as Gallagher's index is now the default.